process/TI-Gives+Gets.md

```diff
@@ -0,0 +1,87 @@
+ * Provides quarterly updates to the TAC on technical vision and progress on vision.
+ * TI will have a [SECURITY.md](http://security.md/) that describes how the Project manages vulnerabilities, or more broadly how the OSSF handles vulnerability reports
+
+### Gets/Benefits
+
+ * TI can get assistance with Architecture & Roadmap Alignment
+ * Receives consideration as in-scope for any submission to an OpenSSF-managed conference or event.
+ * TI receives guidance on technical direction from TAC sponsor
+ * Reserved space for project updates in OpenSSF newsletters.
+ * May request infrastructure support from the OpenSSF.
+ * Projects may say they are, "A sandbox project in the OpenSSF" or "An experimental project in the OpenSSF."  Gets a "sandbox" logo that is shared amongst all OpenSSF sandbox TIs.
+ * Communication & Collaboration - OpenSSF mailing list, OpenSSF Slack channel, OpenSSF GitHub, OpenSSF Calendaring / Recording,  OpenSSF Social Media & External Engagement Support
+ * Governance & Administration - TI Charter Development & Review, TI Technical Steering Committee Setup, TI IP & License Review, TI Operations & Maintenance, Technical Support
+
+
+## Incubating level Gives & Gets
+
+### Gives/Requirements
+
+ All requirements of Sandbox must be fulfilled. PR filed to promote group to Incubating stage.
+ * Group has met no less than 5 times within the last calendar quarter
+ * Maintains a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions.  Projects must have a minimum of three maintainers with a minimum of two different organization affiliations, and document the current list of maintainers.
+ * Projects must have defined a contributor guide, which makes it clear how and when contributors should be given increasing responsibilities towards maintainership of the project. (Example guides: Sigstore, AllStar)
+ * Projects should be able to show adoption by multiple parties and adoption's value to the open source community and/or end users (may include adoption of beta/early versions) with the intent to showcase wide adoption by the project's consumers.
+ * TI must have documented, initial group governance.
+ * Maintains a point of contact for vulnerability reports in the security.md
+ * Implements, practices, and refines mature software development and release practices such as following a version schema.
+ * TI Follows security best practices (as recommended by the OpenSSF and others), including passing the OpenSSF Best Practices criteria
+ * Project should be integrating with Scorecards
+ * Begins to establish the appropriate governance that enables its sustainment for potential graduation.
+ * Projects should be Securing Code Repository -> Managing Contributions Commit Signing , Secret Scanning, Code Scanning (OSFUZZ at a minimum) + Self-assessment Should OpenSSF require these if the SCM supports it, especially using Sigstore?
```

Comment:
I'm always hesitant when I see very prescriptive security requirements, as best practices change over time. What if we struck this line and add "... including secret scanning and code scanning" to "TI Follows security best practices..." above?

Reply:
Thanks a good suggestion, do you want to propose that wording change?